css/index.css

```diff
@@ -98,3 +98,36 @@ header a:not(:last-child) {
     align-items: center;
 }
 
+.img-section-2 {

```

Comment:
bad name, we will have the same bg images in other blocks, so class should work for all blocks.

Reply:
there are no repeating images on the main page. What do you mean?

Reply:
images are not the same, but classes will be the same. for all sections. Difference only in url, and for this we can use separate classes